core/src/main/java/org/jruby/util/JRubyClassLoader.java

```diff
@@ -160,6 +166,10 @@ private static String systemTmpDir() {
         return "";
     }
 
+    public List<URL> getTempUrls(){
```

Comment:
Yeah, that makes sense. What about `cachedJarPaths` and `getCachedJarPaths`? Or is that too verbose?